sending-a-simple-message-to-a-service/README.md

```diff
@@ -0,0 +1,325 @@
+# Sending a Simple Message to a Service
+
+## What you'll build
+
+Let's develop a service that allows a user to retrieve a list of doctors based on the doctor's specialization (category). The information about the doctors is retrieved from a separate microservice. 
+
+To implement this use case, you will develop a REST service with a single resource using Visual Studio Code with the Ballerina Swan Lake extension, and then run the service. The resource will receive the user request, retrieve details from the backend service, and respond to the user request with the relevant doctor details.
+
+### Concepts covered
+
+- REST API
+- HTTP client
+
+## Let's get started!
+
+### Step 1: Set up the workspace
+
+Install [Ballerina Swan Lake](https://ballerina.io/downloads/) and the [Ballerina Swan Lake VSCode extension](https://marketplace.visualstudio.com/items?itemName=wso2.ballerina) on VSCode.
+
+### Step 2: Develop the service
+
+Follow the instructions given in this section to develop the service.
+
+1. Create a new Ballerina project using the `bal` command and open it in VSCode.
+
+```bash
+$ bal new sending-a-simple-message-to-a-service
+```
+
+2. Introduce the source code in files with the `.bal` extension (e.g., the `main.bal` file). 
+
+Import the 
+- `ballerina/http` module to develop the REST API and define the client that can be used to send requests to the backend service
+- `ballerina/log` module to log some information for each client request
+
+```ballerina
+import ballerina/http;
+import ballerina/log;
+```
+
+3. Define two [configurable variables](https://ballerina.io/learn/by-example/#configurability) for the port the listener should listen on and the URL of the backend service.
+
+```ballerina
+configurable int port = 8290;
+configurable string healthcareBackend = "http://localhost:9090/healthcare";
+```
+
+4. Define an [`http:Client` client](https://ballerina.io/learn/by-example/#http-client) to send requests to the backend service.
+
+```ballerina
+final http:Client queryDoctorEP = check initializeHttpClient();
+
+function initializeHttpClient() returns http:Client|error => new (healthcareBackend);
+```
+
+The argument to the `new` expression is the URL for the backend service.
```

Comment:
```suggestion
The argument to the `new` expression is the URL for the backend service.
```
Shall we have this as in `info` section?

Reply:
No, this follows a pattern of describing aspects of the code after the snippet, under each point. For other tutorials there will be more details like this.